inc/CrvHitInfoReco.hh

```diff
@@ -0,0 +1,39 @@
+#ifndef CrvHitInfoReco_hh
+#define CrvHitInfoReco_hh
+
+#include "Offline/DataProducts/inc/GenVector.hh"
+#include "CLHEP/Vector/ThreeVector.h"
+#include <vector>
+#include "Rtypes.h"
+
+namespace mu2e
+{
+
+  struct CrvHitInfoReco   //information about a cluster of CRV coincidence triplets
+  {
+    Int_t               sectorType =-1;   //CRV sector type
+    XYZVectorF          pos; //average position of counters
```

Comment:
a Float_t  time=-1; could be added for the average hit time.

Reply:
Will do.  I presume it should be filled with cluster.GetAvgHitTime()

Reply:
Yes, I made a comment there. There are also a few other places where the average hit time should be added or changed from the earliest hit time.